docs/CODE_GUIDELINES.md

```diff
@@ -341,28 +358,49 @@ int const * const someIntPointer;
 
 ### 5.4. Initialization
 
-Make sure that variables are initialized appropriately at declaration or soon afterwards. This is especially important for fundamental type variables that do not have any constructor. Zero-initialize with `{}`.
+Make sure that variables are initialized appropriately at declaration or soon afterwards. This is especially important for fundamental type variables that do not have any constructor to zero-initialize with `{}`.
```

Comment:
Can you rephrase the second sentence? I don't understand what it is trying to say after the change (grammar also feels off).

Reply:
typo i will revert the change